### DISCUSSION TITLE
Server API cannot be loaded.

### DISCUSSION THREAD

**Discussion by siud112** [SafeExamBrowser/seb-server#79] [Q&A]
Hi SEB team,
unfortunately I fail to connect to the SEB server with the mobile app (iPad).

In the app under more information -> Admin -> Settings -> SEB Server... activate connect to SEB Server and enter the address of my server under the server URL.

Always get the error message on the iPad:
Connection to SEB Server failed. Server API cannot be loaded.

what can i do?

thank you for your help!

**Comment by anhefti** [comment]
Hi, 

To connect your SEB client with an SEB Server you need a valid Connection Configuration form that SEB Server. You can download such a Connection Configuration from the SEB Server:

![image](https://github.com/SafeExamBrowser/seb-server/assets/39078777/5178c178-bdf7-41dd-aea6-ed2691d23d42)
![image](https://github.com/SafeExamBrowser/seb-server/assets/39078777/42fb885d-56ae-4b4f-b883-4ca0dc6137c1)

Or if you don't have access to a SEB Server you may ask your exam administrator to give you the valid SEB Server configuration.
Once you have the configuration you can just load it with your SEB client.

Best regards

**Comment by siud112** [comment]
Hi,
thanks for the answer. Unfortunately, the error with the API message still comes up.

- maybe add something in the server settings?

best regards

**Reply by danschlet** [reply]
The settings in SEB for iOS and macOS only show the SEB Server main URL, mainly to indicate to users that this is a configuration file to connect to SEB Server. You cannot configure an SEB Server connection in the SEB Client only, you always have to use a connection configuration correctly exported from SEB Server.

**Comment by siud112** [comment]
unfortunately that doesn't work for me.
- I created a new connection configuration on the server -> exported it -> opened it in the app...API error (API cannot be loaded)

- may still have to be in the server config (application-*.properties)
In the file I only adjusted the setting: sebserver.webservice.http.external.*

**Reply by anhefti** [reply]
Hi, 

Make sure the sebserver.webservice.http.external.* settings are correct set. You can check this on SEB Server startup within the log-output:
![image](https://github.com/SafeExamBrowser/seb-server/assets/39078777/ea934068-1776-4f7d-b298-07d1d9ec4db5)

If this is not correct do also take a look into your docker-compose.yml configuration file and if the settings are there, you have to set it in the docker-compose.yml file:

![image](https://github.com/SafeExamBrowser/seb-server/assets/39078777/7c211744-b528-4e3c-bd53-8ba941883c94)

If this also don't work, can you please attach the connection configuration file from SEB Server here, so we can have a look at it.

**Comment by siud112** [comment]
oh no, a typo in the docker-compose.yml
Everything works now!

Thanks very much!

**Comment by anhefti** [comment]
Glad the hear, best!
